sqlx-mysql/src/options/parse.rs

```diff
@@ -176,3 +182,47 @@ fn it_returns_the_parsed_url() {
 
     assert_eq!(expected_url, opts.build_url());
 }
+
+#[test]
+fn it_parses_connection_attributes_false() {
+    let url = "mysql://username:password@hostname:5432/database?connection-attributes=false";
```

Comment:
I forgot about this case. We probably want this to ensure that no attributes are sent, even if the application adds them. It wouldn't make much sense otherwise.

Sorry for the mislead, but the enum approach was probably better to handle this. However, instead of matching on the enum in the `MySqlOptions` methods, the pattern matching should be handled by methods on `Attributes`.

Reply:
Currently the option `connection-attributes=false` does not have any use any more it is a remnant from the time where the default was das the library would send "_client_version" and "_client_name" by default.

Yeah if thats desired, I can surely implement that, I find it a little bit confusing that this will then behave differently then all other connection settings which can be overwritten in code.

Also it is currently not possible to both send the "default" attributes and add custom attributes via the connection string. I'm not sure how to best solve that and still stay consistent with the behavior these connection-attributes work in other implementations.